src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/AsyncGoalEventService.kt

```diff
@@ -23,18 +23,45 @@ class AsyncGoalEventService(
   override fun goalCreated(prisonNumber: String, createdGoal: Goal) {
     runBlocking {
       log.debug { "Goal created event for prisoner [$prisonNumber]" }
-      val timelineEvent = timelineEventFactory.goalCreatedTimelineEvent(createdGoal)
-      launch { timelineService.recordTimelineEvent(prisonNumber, timelineEvent) }
-      launch { telemetryService.trackGoalCreateEvent(createdGoal) }
+      launch { recordCreateGoalTimelineEvent(prisonNumber = prisonNumber, createdGoal = createdGoal) }
```

Comment:
Re: our conversation yesterday @mattwills-moj-digital ; I've refactored into private methods so there is no risk of us using the timeline events for other types of event tracking (ie. App Insights)